client/devdocs/gutenberg-components/examples.json

```diff
@@ -0,0 +1,90 @@
+[
+	{ "component": "Autocomplete" },
+	{ "component": "BaseControl" },
```

Comment:
Would it be possible to generate this list on the fly? Given that it is README based it quickly might become outdated. Unless you have a plan how to prevent regressions.

Reply:
This seems wise. I think we could perhaps leave this until a follow-up PR, on the proviso that we'll actually follow up :)

Reply:
The main reason for having this file is to avoid rendering deprecated components like `withContext` or `withAPIData`. It's true we can autogenerate the rest of the data, but I don't know if we have some way of knowing if a component is deprecated or not.